pmm-app/src/pmm-dbaas/components/DBCluster/DBCluster.test.tsx

```diff
@@ -24,12 +24,8 @@ describe('DBCluster::', () => {
   it('renders correctly with failed status', () => {
     const root = mount(<DBCluster kubernetes={kubernetesStub} />);
 
-    expect(
-      root
-        .find('[data-qa="cluster-status-failed"]')
-        .at(0)
-        .prop('className'),
-    ).toContain('failed');
+    expect(root.find('[data-qa="cluster-progress-bar"]').length).toBeGreaterThan(0);
+    expect(root.find('[data-qa="cluster-status-error-message"]').length).toBeGreaterThan(0);
```

Comment:
Can we use `dataQa` here?

Reply:
It was an old file so had to change in the rest of the file too, but done!

Reply:
👍